tests/src/test/java/alluxio/client/cli/fs/command/LeaderCommandIntegrationTest.java

```diff
@@ -33,7 +33,15 @@ public void leader() {
   public void leaderAddressNotAvailable() throws Exception {
     mLocalAlluxioCluster.stopMasters();
     mFsShell.run("leader");
-    String expected = "The leader is not currently serving requests.\n";
-    Assert.assertEquals(expected, mErrOutput.toString());
+    String expected = "The leader is not currently serving requests.";
+    Assert.assertEquals(expected, lastLine(mErrOutput.toString()));
+  }
+
+  private String lastLine(String output) {
+    String[] lines = output.split("\n");
+    if (lines.length > 0) {
+      return lines[lines.length - 1];
+    }
+    return "";
```

Comment:
Eh it's only a test, but maybe a better way to retrieve the last line is to do

```java
int idx = output.lastIndexOf("\n");
idx = idx == -1 ? 0 : idx+1;
return output.substring(idx);
```

`output.substring(output.indexOf("\n")

Reply:
Actually come to think you could just do 

```java

Reply:
After consideration, I think`String.contains()` should suffice.